sdk/turing/router/config/router_ensembler_config.py

```diff
@@ -160,16 +160,14 @@ def standard_config(self, standard_config: Union[EnsemblerStandardConfig, Dict])
             self._standard_config = standard_config
         elif isinstance(standard_config, dict):
             openapi_standard_config = standard_config.copy()
-            openapi_standard_config["experiment_mappings"] = (
-                [
+            openapi_standard_config["experiment_mappings"] = None
+            if "experiment_mappings" in standard_config and standard_config["experiment_mappings"] is not None:
+                openapi_standard_config["experiment_mappings"] = [
                     turing.generated.models.EnsemblerStandardConfigExperimentMappings(
                         **mapping
                     )
                     for mapping in standard_config["experiment_mappings"]
                 ]
-                if openapi_standard_config is None
-                else None
```

Comment:
Just to make sure I understood what was happening previously, was `openapi_standard_config["experiment_mappings"]` set as `None` previously because `openapi_standard_config` is non-nil (in the case of the router that Roman was using)?

Reply:
Correct. Experiment mappings should in fact have been non-nil if openapi_standard_config["experiment_mappings"] (or standard_config["experiment_mappings"]) was non-nil. Instead, it was being set to None.